CHANGELOG.md

```diff
@@ -1,3 +1,20 @@
+# 2.0.0
+
+Breaking changes:
```

Comment:
@tatey since you've been using & contributing to this library, would love to hear if you have any concerns / suggestions around these changes :)

Reply:
Thanks for considering me. I've only read the CHANGELOG and from what's described in here I don't have any concerns or suggestions at this time.